snuba/replacers/errors_replacer.py

```diff
@@ -215,6 +215,14 @@ def process_message(
                     "Skipping replacement for project",
                     extra={"project_id": processed.get_project_id(), "data": message},
                 )
+                metrics.increment(
+                    "bypass_projects",
```

Comment:
overall fine, but let's rename this to something more descriptive: "replacement_message_skipped"